pkgs/os-specific/linux/kernel/htmldocs.nix

```
@@ -10,35 +10,6 @@
 , which
 }:
 
-let
-  py = python3.override {
-    packageOverrides = final: prev: rec {
-      docutils_old = prev.docutils.overridePythonAttrs (oldAttrs: rec {
```

This should have been
```suggestion
      docutils = prev.docutils.overridePythonAttrs (oldAttrs: rec {
```
to prevent the error.

---

I did this on purpose to prevent many hundred rebuilds while doing local testing but at the time we didn't have the check in place that caught this now.